pmm-app/src/pmm-add-instance/components/AddRemoteInstance/AddRemoteInstance.service.tsx

```diff
@@ -0,0 +1,96 @@
+import { apiRequestManagement } from 'shared/components/helpers/api';
+import { TrackingOptions } from './AddRemoteInstance.types';
+
+class AddRemoteInstanceService {
+  static async addMysql(body) {
+    return apiRequestManagement.post<any, any>('/MySQL/Add', body);
+  }
+
+  static async addPostgresql(body) {
+    return apiRequestManagement.post<any, any>('/PostgreSQL/Add', body);
+  }
+
+  static async addProxysql(body) {
+    return apiRequestManagement.post<any, any>('/ProxySQL/Add', body);
+  }
+
+  static async addMongodb(body) {
+    return apiRequestManagement.post<any, any>('/MongoDB/Add', body);
+  }
+
+  static async addRDS(body) {
+    return apiRequestManagement.post<any, any>('/RDS/Add', body);
+  }
+
+  static addRemote(type, data) {
+    switch (type) {
+      case 'MongoDB':
```

Comment:
Can use DATABASE_LABELS from core/constants.ts here. Just add one more for ProxySQL and another type.

Reply:
Thanks, done